app/root.tsx

```diff
@@ -26,5 +29,17 @@ export function Layout({ children }: { children: React.ReactNode }) {
 }
 
 export default function App() {
-  return <Outlet />;
+  const domain = import.meta.env.VITE_DOMAIN;
+  const clientID = import.meta.env.VITE_CLIENT_ID;
+  const redirect_uri = import.meta.env.VITE_REDIRECT_URI;
```

Comment:
I think it would be an improvement for these to come in from `context.cloudflare.env` like `context.cloudflare.env.ALIGNMENT_FEED_BASE_URL`, so we support reading it in from the Cloudflare Pages config on production deployments.